data/cloudstack.yml

```diff
@@ -1,11 +1,11 @@
 versions:
-  main: 4.14.0.0
+  main: 4.15.0.0
   main_islts: yes
-  main_docs_url: http://docs.cloudstack.apache.org/en/4.14.0.0/
-  branch: 4.14
-  lts: 4.14.0.0
+  main_docs_url: http://docs.cloudstack.apache.org/en/4.15.0.0/
+  branch: 4.15
+  lts: 4.15.0.0
   lts_docs_url: http://docs.cloudstack.apache.org/en/latest/installguide/index.html
-  ltsbranch: 4.14
+  ltsbranch: 4.15
   archives:
     - '4.14':
        - 4.14.0.0  
```

Comment:
@DaanHoogland can you add 4.14.1.0 here as well. It seem the master branch is not updated, but asf-site is.

Reply:
not sure what the semantics are, can you check if this is what you mean @rhtyd ? (I'll run middleman as well to check)